products/platform/documentation-site/research/search-design-doc.md

```diff
@@ -1,15 +1,157 @@
+
+### Open Questions and Risks
+
+- Should we index private documentation sources? If so, what private documentation sources can we expose to public users, if any?
+- Should the landing page be public only, or should it also support authenticated users?
+- How often should we reindex our documentation sources? For perspective, Algolia's open source crawler runs every 24 hours.
+- What should the URL of the documentation landing page be? Since we have `design.va.gov`, does something like `docs.va.gov` make sense?
+- What is the current opportunity cost of building a custom solution? What are we choosing to delay right now by building a custom solution?
+- What is the future opportunity cost of maintaining a custom solution? What will we have to delay in the future by maintaining a custom solution?
+- What is the key person risk? How disruptive would it be if the author(s) of the custom solution were temporarily/permanently unavailable?
+- What is the Not-Invented-Here risk? How likely is that we're reinventing the wheel?
+- What is the inexperience risk? Have any of the team members involved ever built/deployed/maintained this type of system before?
+- It's possible that work estimates are materially inaccurate.
+- What is the documentation debt risk? At least one person interviewed during the discovery sprint said that we need a solution ASAP. Is there a risk that people are deferring documentation in the present until the custom documentation solution is shipped?
+
+### Work Estimates
+
+1. Provision and configure data store to house records that will be returned from the Search API. - 1 week
+
+- I estimate one week because I suspect we'll either need to get approval to use Algolia's hosted API or get approval to provision a custom data store.
+
+1. Configure crawler to seed the initial data store. - 2 days
+1. Run the crawler to populate the data store. - 2 day
+1. Build a landing page that will contain a search input that submits requests to the Search API. - 2 day
+1. Deploy the landing page to a publicly available location. - 2 day
+
+- I estimate this may take longer if we need to need to coordinate with other teams.
+
+### Alternatives
+
+#### Ready-made alternatives
+
+- [Slab](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/platform/documentation-site/research/discovery-sprint-12-2019/technology-discovery.md#slab) supports multi-repo search, but was considered too expensive (about \$24k/year for the current number of users in the #general channel of the DSVA Slack workspace).
```

Comment:
Considered too expensive by whom? Citation?

Reply:
Let's update this to be more reflective of the nuance of our decision — we (as a discovery sprint team — this can be cited to discovery sprint documentation) did not recommend Slab _as an MVP_ for a variety of reasons:
- Price
- User-based access (you could share public links to documentation, but as far as we could tell to navigate the site within Slab would require a Slab account, or an external, public, custom homepage anyway). Relatedly, does not meet users in their existing workflow.
- Not an MVP, requires "all or nothing" implementation
- No guarantee that it would stay accessible on VA network

We don't need to mention all of those in the design doc, but saying that it was exclusively about price is missing the nuance. Also, should we mention that Slab is in fact still on the table as a post-MVP solution? I don't think that's necessary to include but not sure.

Also, I do agree that it's good to bring up off the shelf solutions in this design doc, since they're very relevant to the discussion, but it's worth noting that we're not comparing apples to apples. Slab has a search functionality (Algolia, in fact), but it is also a complete documentation site solution. It's an alternative that would replace custom search, yes, but it's not a direct search-to-search comparison. Does that make sense? If so, that should be clarified in the design doc so that everyone understands what it is.

Reply:
That makes sense. Updated in https://github.com/department-of-veterans-affairs/va.gov-team/pull/5248/commits/4207adb63eda693989ab1ce7a45a18f6cf0a57b3